manifests/init.pp

```diff
@@ -231,7 +231,7 @@
       path    => "${confdir}/puppet.conf",
       section => 'main',
       setting => 'hiera_config',
-      value   => $hiera_yaml,
+      value   => '$confdir/hiera.yaml',
```

Comment:
Wouldn't this be the right value after the other change in the commit? Seems like this would hard code a dynamic value.